recipes-wpe/wpewebkit/wpewebkit.inc

```diff
@@ -108,7 +108,7 @@ EXTRA_OECMAKE += "\
     -DCMAKE_BUILD_TYPE=Release \
     -DPORT=WPE \
     -G Ninja \
-    -DPYTHON_EXECUTABLE=${PYTHON_EXECUTABLE} \
+    -DPYTHON_EXECUTABLE=${PYTHON} \
```

Comment:
In this case we need to handle both PYTHON and PYTHON_EXECUTABLE, as inherit is done dynamically based on yocto version (https://github.com/WebPlatformForEmbedded/meta-wpe/pull/557/files?file-filters%5B%5D=.inc#diff-47815a43578d39ac922573a4725079768d0182ee9a13aacd7f3c99a713326c0cL24)

Reply:
I don't think we do, because `PYTHON` is exported regardless of the version.

`pythonative.bbclass`

```

inherit python-dir

PYTHON="${STAGING_BINDIR_NATIVE}/${PYTHON_PN}-native/${PYTHON_PN}"
# PYTHON_EXECUTABLE is used by cmake
PYTHON_EXECUTABLE="${PYTHON}"
EXTRANATIVEPATH += "${PYTHON_PN}-native"
DEPENDS += " ${PYTHON_PN}-native "

```

`python3native.bbclass`

```
PYTHON_BASEVERSION = "3.5"

inherit python-dir

PYTHON="${STAGING_BINDIR_NATIVE}/${PYTHON_PN}-native/${PYTHON_PN}"
EXTRANATIVEPATH += "${PYTHON_PN}-native"
DEPENDS += " ${PYTHON_PN}-native "

```